addons/project/migrations/12.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,145 @@
+project      / project.project          / percentage_satisfaction_task (integer): previously in module rating_project
+project      / project.project          / portal_show_rating (boolean)  : previously in module website_rating_project
+project      / project.project          / rating_request_deadline (datetime): previously in module rating_project
+project      / project.project          / rating_status (selection)     : previously in module rating_project
+project      / project.project          / rating_status_period (selection): previously in module rating_project
+project      / project.task             / rating_count (integer)        : previously in module rating_project
+project      / project.task             / rating_ids (one2many)         : previously in module rating_project
+project      / project.task             / rating_last_feedback (text)   : previously in module rating_project
+project      / project.task             / rating_last_image (binary)    : previously in module rating_project
+project      / project.task             / rating_last_value (float)     : previously in module rating_project
+project      / project.task.type        / auto_validation_kanban_state (boolean): previously in module rating_project
+project      / project.task.type        / rating_template_id (many2one) : previously in module rating_project
+# NOTHING TO DO: rating_project and website_rating_project merged to project
+
+project      / project.task             / remaining_hours (float)       : now a function
+# NOTHING TO DO
+
+project      / res.company              / project_time_mode_id (many2one): relation is now 'uom.uom' ('product.uom')
+# NOTHING TO DO
+
+---XML records in module 'project'---
+NEW digest.digest: digest.digest_digest_default (noupdate)
+NEW digest.tip: project.digest_tip_project_0
+NEW ir.actions.act_window: project.rating_rating_action_project_report
+NEW ir.actions.act_window: project.rating_rating_action_task
+NEW ir.actions.act_window: project.rating_rating_action_view_project_rating
+DEL ir.actions.act_window: project.action_project_task_user_tree_filtered
+DEL ir.actions.act_window: project.project_task_action_activity
+DEL ir.actions.act_window: project.project_task_actions_act_window_merge_tasks
+DEL ir.actions.act_window: rating_project.rating_rating_action_project_report
+DEL ir.actions.act_window: rating_project.rating_rating_action_task
+DEL ir.actions.act_window: rating_project.rating_rating_action_view_project_rating
+DEL ir.actions.act_window: rating_project.rating_rating_action_view_project_rating_task
+# NOTHING TO DO
+
+DEL ir.attachment: project.msg_task_data_14_attach (noupdate)
+DEL ir.attachment: project.msg_task_data_8_attach (noupdate)
+# DONE: post-migration: Deleted by xml_id
+
+NEW ir.cron: project.ir_cron_rating_project (noupdate)
+DEL ir.cron: rating_project.ir_cron_rating_project
+# DONE: pre-migration: renamed xml ids
```

Comment:
No need. It's already renamed on the merge modules operation. Maybe you have to switch noupdate status, as previously it seems that it was noupdate="0".

Reply:
Where/how to switch?

Reply:
`openupgrade.update_module_names` already perform this. Check the code.

Reply:
What I'm missing is the merge operation in apriori.py...

Reply:
https://github.com/OCA/OpenUpgrade/blob/12.0/odoo/addons/openupgrade_records/lib/apriori.py#L17

Reply:
OK, it hasn't been done in this PR then, but as said, you have to remove this as it's already done.